src/components/Vote/AfterVoteList/index.tsx

```diff
@@ -1,34 +1,50 @@
-import { useEffect, useState } from "react";
-
 import { IcCheck1 } from "../../../asset/icon";
-import { voteContent, voteTitle } from "../../../core/vote/voteContent";
+import { real } from "../../../core/api/vote";
 import { St } from "./style";
 
-export default function AfterVoteList(props: any) {
-  const { setIsVoted, setIsSuccess, currentIndex } = props;
+type propsType = {
+  setIsVoted: React.Dispatch<React.SetStateAction<boolean>>;
+  setIsSuccess: React.Dispatch<React.SetStateAction<boolean>>;
+  currentIndex: number;
+  ballotTopic: any;
+  setIsPosted: React.Dispatch<React.SetStateAction<boolean>>;
+};
+export default function AfterVoteList(props: propsType) {
```

Comment:
any는 제가 고칠게욥

props 에 대한 컨벤션 지켜주세요!

```suggestion
interface AfterVoteListProps = {
  setIsVoted: React.Dispatch<React.SetStateAction<boolean>>;
  setIsSuccess: React.Dispatch<React.SetStateAction<boolean>>;
  currentIndex: number;
  ballotTopic: any;
  setIsPosted: React.Dispatch<React.SetStateAction<boolean>>;
};
export default function AfterVoteList(props: AfterVoteListProps) {
```

Reply:
이거 바로 commit suggestion 해도 될 것 가ㅏㅌ아욥